composer.json

```diff
@@ -27,16 +27,20 @@
         "classmap": ["src/Omnipay/Omnipay.php"]
     },
     "require": {
-        "php": ">=5.3.2",
-        "guzzle/guzzle": "~3.9",
-        "symfony/http-foundation": "~2.1"
+        "php": ">=5.5.9",
+        "symfony/http-foundation": "^2.1|^3.0",
```

Comment:
Yes, it should be removed. I've removed the ParameterBag now, but not yet the RedirectResponse.